### ISSUE TITLE
Page doesn't load in IE 11

### ISSUE THREAD

**Issue by julienlegault** [AuburnACM/auacm#120]
As title states, the page doesn't load in Internet Explorer 11, I'm not sure why but it is unable to load.

**Comment by BrandonLMorris** [comment]
Any JS logs?

**Comment by julienlegault** [comment]
No, it's not throwing any errors. The problem is it isn't ever loading the body, all it loads in the <app-root>loading... </app-root> and the scripts, but never resolves any of it.

**Comment by julienlegault** [comment]
Also the site crashes on all older versions of IE, but I feel like thats not really a problem.

**Comment by BrandonLMorris** [comment]
You're not running locally, correct? This is happening when you go to auacm.com in IE11: the production site, right?

**Comment by julienlegault** [comment]
Yeah, it's the production site.

**Comment by Mac-Genius** [comment]
It does not load in IE11 because IE11 does not have support for Javascript es6. We would need to add a polyfill es6 library that would add support. You can learn about the fix [here](https://angular.io/docs/ts/latest/guide/browser-support.html).
